File: Sketch.cs
using System;

namespace MS.Paint2
{
    public class Sketch
    {
        public Sketch()
        {

        }

        public Sketch(string name, string imageData, DateTime? dateCreated)
        {
            Name = name;
            ImageData = imageData;
            DateCreated = dateCreated;
        }

        public string Name { get; set; }
        public string ImageData { get; set; }

        public DateTime? DateCreated { get; set; }
    }
}

File: ClientApp/src/services/Sketch.ts
//Domain object Sketch.
export class Sketch {
    name: string;

    imageData: string;

    dateCreated: Date;

    constructor(name: string, imageData: string, dateCreated: Date) {
        this.name = imageData;
        this.imageData = imageData;
        this.dateCreated = dateCreated;
    }
}